janitor/functions/min_max_scale.py

```diff
@@ -84,6 +98,7 @@ def min_max_scale(
     :param df: A pandas DataFrame.
     :param feature_range: (optional) Desired range of transformed data.
     :param column_name: (optional) The column on which to perform scaling.
+    :param entire_data: (bool) Scale the entire data if Ture.

```

Comment:
Does mkdoc have any anochers like `versionadded`, `deprecated` like [sphinx](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#paragraph-level-markup=)

Reply:
I don't think `mkdocs` has this functionality. We could simply add that in as part of the text, maybe?

Reply:
Maybe we could add the changing list at the end like the following?

```python
"""
    ...

    :param df: A pandas DataFrame.
    :param feature_range: (optional) Desired range of transformed data.
    :param column_name: (optional) The column on which to perform scaling.
    :param entire_data: (bool) Scale the entire data if Ture.
    :returns: A pandas DataFrame with scaled data.
    :raises ValueError: if `feature_range` isn't tuple type.
    :raises ValueError: if the length of `feature_range` isn't equal to two.
    :raises ValueError: if the element of `feature_range` isn't number type.
    :raises ValueError: if `feature_range[1]` <= `feature_range[0]`.

    Changed in version 0.24.0: Deleted "old_min", "old_max", "new_min", and "new_max" options.
    Changed in version 0.24.0: Added "feature_range", and "entire_data" options.
"""
```
![image](https://user-images.githubusercontent.com/25895405/172081430-dee508a4-4a8c-47fb-b089-fb96b7b772f1.png)